src/games/strategy/engine/framework/mapDownload/DownloadRunnable.java

```diff
@@ -52,26 +59,45 @@ public void setError(final String error) {
 
   @Override
   public void run() {
```

Comment:
Any reason we need to be able to use a local file? Wouldn't we always be either pointing to SF or github?